app/components/c-product-detail.vue

```diff
@@ -26,7 +26,10 @@
     <section :class="b('bottom')">
 
       <div :class="b('main', { area: 'bottom' })">
-        <div :class="b('details')"> details</div>
+        <div v-if="product.description" :class="b('details')">
+          <e-heading underline tag-name="h2" color="gray">{{ $t('c-product-detail.product-description') }}</e-heading>
+          <div :class="b('description')" v-html="product.description"></div>
+        </div>
```

Comment:
General: How is the concept about the product-detail? Which components are separate, single components which are integrated in the `c-product-detail` and which are directly implemented here?

Reply:
@claudio147 @geroldboehler Hey guys, just picked-up this conversation about the product detail. As Patric explained to me, this component is only a wrapper for all the other components e.g. gallery, accordion etc. As you can see I already tried to name the different sections. Let's talk with Patric about that.